docker/Dockerfile

```diff
@@ -113,9 +113,12 @@ RUN SCRATCH_DIR=/scratch && mkdir -p ${SCRATCH_DIR} && cd ${SCRATCH_DIR} && \
     make -j${NPROCS} && make install && \
     rm -rf ${SCRATCH_DIR}
 
+# Workaround for Kokkos to find libcudart
+ENV LD_LIBRARY_PATH=/usr/local/cuda/targets/x86_64-linux/lib:${LD_LIBRARY_PATH}
```

Comment:
Do we still need that workaround?

Reply:
I think we still need that for the release, but it might be fixed by https://github.com/kokkos/kokkos/pull/2712. We can just check again to be sure.

Reply:
doing that now

Reply:
Seems we still need it.

Reply:
Dropped the last commit again.

Reply:
Fixed in https://github.com/kokkos/kokkos/pull/2722 which is not part of the release unfortunately.